### DISCUSSION TITLE
Numeric Keytboard whit " . "

### DISCUSSION THREAD

**Discussion by SamuelPelletierEvraire** [unoplatform/uno#11609] [Q&A]
Hi everyone, I am currently looking for a way to make a numeric keyboard appear with a dot in order to be able to write decimal numbers. Is there a way to do this?

**Comment by jeromelaban** [comment]
The [`TextBox.InputScope`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.textbox.inputscope?view=winrt-22621) property is generally built for this scenario. If you expect the on-screen keyboard (on devices that do support it) you can use this property when a TextBox is available.
